src/main/java/com/exasol/repository/AbstractGitHubGitBranchContent.java

```diff
@@ -32,9 +32,8 @@ private GHBranch getBranchByName(final String branch) {
         try {
             return this.repository.getBranch(branch);
         } catch (final IOException exception) {
-            throw new GitHubException(
-                    "Cannot find a branch '" + branch + "'. Please check if you specified a correct branch.",
-                    exception);
+            throw new GitHubException("E-REP-AGHGBC-1: Cannot find a branch '" + branch
```

Comment:
```suggestion
            throw new GitHubException("E-GH-REP-1: Cannot find a branch '" + branch
```
Short tags should reflect modules, not classes. So make it a little bit more coarse please. Downside is that you need to keep track of the error number across classes of a module. A module can span multiple Java packages if necessary.

I am struggling to find a good short tag.